doc/source/deframer_packet_processor.rst

```diff
@@ -0,0 +1,122 @@
+AIT Deframer/Packet Processor
```

Comment:
Can we update this so it's linked in the "table of contents". Otherwise it's a bit difficult to navigate to.